Integration/Iterator/AbstractIterator.php

```diff
@@ -23,7 +23,7 @@ abstract class AbstractIterator implements \Iterator
     /**
      * @var LoggerInterface
      */
-    private $logger;
+    protected $logger;
```

Comment:
Makes sense to switch to \Psr\Log\LoggerAwareTrait then

Reply:
done